app/src/main/java/ca/bc/gov/bchealth/ui/setting/ProfileFragment.kt

```diff
@@ -85,8 +86,9 @@ class ProfileFragment : Fragment(R.layout.fragment_profile) {
                         return@collect
                     } else {
                         initClickListeners()
-                        binding.layoutProfile.isVisible = it.isLoggedIn
-                        binding.layoutLogin.isVisible = !it.isLoggedIn
+                        val loginStatus = it.loginSessionStatus == LoginSessionStatus.ACTIVE
```

Comment:
As you are checking that login session is active or expired, I suggest to rename loginStatus variable to isLoginStatusActive or any other suitable variable name which increases readability.